Documentation/troubleshooting.rst

```diff
@@ -378,6 +378,47 @@ Below is an approximate list of the kind of information in the archive.
 * ``cilium service list``
```

Comment:
Typo in commit: "Make k8s-get-cilium-pod.shavailable" --> "Make k8s-get-cilium-pod.sh available"